components/post_view/post_body_additional_content/post_body_additional_content.test.jsx

```diff
@@ -66,4 +69,27 @@ describe('components/post_view/PostBodyAdditionalContent', () => {
         expect(props.actions.toggleEmbedVisibility).toHaveBeenCalledTimes(1);
         expect(props.actions.toggleEmbedVisibility).toBeCalledWith('post_id_1');
     });
+
+    test('image link should go image through proxy in preview modal', () => {
```

Comment:
Minor typo correction? `image link should go *image* through proxy in preview modal` ... remove extra `image`?